src/Surfnet/StepupMiddleware/ApiBundle/Configuration/Projector/InstitutionConfigurationOptionsProjector.php

```diff
@@ -35,9 +34,17 @@
      */
     private $repository;
 
-    public function __construct(InstitutionConfigurationOptionsRepository $repository)
-    {
-        $this->repository = $repository;
+    /**
+     * @var AllowedSecondFactorRepository
+     */
+    private $allowedSecondFactorRepository;
+
+    public function __construct(
+        InstitutionConfigurationOptionsRepository $repository,
+        AllowedSecondFactorRepository $allowedSecondFactorRepository
+    ) {
+        $this->repository                    = $repository;
```

Comment:
since we now have two repo's, perhaps both properties should have a good name rather than one repository and one specificlyNamedOnPurposeToIdentityWhichItIsRepository 😛